sktime/forecasting/base/_base.py

```diff
@@ -2540,6 +2540,152 @@ def _get_columns(self, method="predict", **kwargs):
 BaseForecaster._init_dynamic_doc()
 
 
+class BaseGlobalForecaster(BaseForecaster):
+    """Base global forecaster template class.
+
+    This class is a temporal solution, might be merged into BaseForecaster later.
+
+    The base forecaster specifies the methods and method signatures that all
+    global forecasters have to implement.
+
+    Specific implementations of these methods is deferred to concrete forecasters.
+
+    """
+
+    _tags = deepcopy(BaseForecaster._tags)
+    _tags["object_type"] = ["global_forecaster", "forecaster"]
```

Comment:
do this by setting the tag dict explicitly

Reply:
changed to setting the tag dict explicitly